golangci.yml

```diff
@@ -1,6 +1,10 @@
 run:
   deadline: 6m
 
+linters-settings:
```

Comment:
Why need this? In most case we should make changes to our code, not the lint configuration

Reply:
The default line-length set of lint is 120 characters for one code line, while after my change of the TopicLevelPolicyNotEnabledError msg `ut: "[✖] code: 405 reason: Topic level policy is disabled, please enable broker configs of systemTopicEnabled and topicLevelPoliciesEnabled",` is almost 145 characters. 
If we need to keep the 120 characters line-length set, i think i need remove the config names in TopicLevelPolicyNotEnabledError.

Reply:
you can split the strings into two lines.  for example:
```
"[✖] code: 405 reason: Topic level policy is disabled, please enable broker "
+ "configs of systemTopicEnabled and topicLevelPoliciesEnabled"
```

Reply:
OK, I see.

Reply:
Upated in https://github.com/streamnative/pulsarctl/pull/439/commits/ef2b05fef7a402f415d5ef4d94bbe3dbddfc3689